### PR TITLE
update security-compatibility-with-mysql.md

### PR DESCRIPTION
**Pull request by tiancaiamao**
### What is changed, added or deleted? (Required)

Tiny update security-compatibility-with-mysql.md

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### CODE REVIEW

**Comment by tiancaiamao** [comment]
PTAL @kissmydb @jackysp

File: security-compatibility-with-mysql.md [hunk 1]
```diff
@@ -8,10 +8,9 @@ aliases: ['/docs-cn/dev/reference/security/compatibility/']
...
 除以下功能外，TiDB 支持与 MySQL 5.7 类似的安全特性。
 
-- 仅支持 `mysql_native_password` 身份验证方案。
+- 仅支持 `mysql_native_password` 密码验证或证书验证登陆方案。
 - 不支持外部身份验证方式（如 LDAP）。
 - 不支持列级别权限设置。
 - 不支持密码过期，最后一次密码变更记录以及密码生存期。[#9709](https://github.com/pingcap/tidb/issues/9709)
 - 不支持权限属性 `max_questions`，`max_updated`，`max_connections` 以及 `max_user_connections`。
 - 不支持密码验证。[#9741](https://github.com/pingcap/tidb/issues/9741)
-- 不支持透明数据加密（TDE）。
```
Comment (kissmydb) [inline security-compatibility-with-mysql.md:17]: 貌似需要一个单独的 TDE 的说明文档

Comment (lysu) [reply security-compatibility-with-mysql.md:17]: https://github.com/pingcap/docs-cn/pull/3325/files 安全部分有，数据加密和传输加密放一起

**Pull request merged**